dht/Pathfinder.c

```diff
@@ -363,6 +363,22 @@ static Iface_DEFUN handlePong(struct Message* msg, struct Pathfinder_pvt* pf)
     return NULL;
 }
 
+static Iface_DEFUN syncNode(struct Message* msg, struct Pathfinder_pvt* pf)
+{
+    uint8_t addr[16];
+    Message_pop(msg, addr, 16, NULL);
+    Assert_true(!msg->length);
+    {
```

Comment:
(informational)
```if (Defined(Log_DEBUG)) {``` will work here. Note: you need to ```#import "util/Defined"```
and don't ask how it works :)